lib/ripper-tags.rb

```diff
@@ -164,6 +167,18 @@ def self.option_parser(options)
   end
 
   def self.process_args(argv, run = method(:run))
+    1.times do # because `redo` works on yielded blocks only
+      optsfile = catch(:reload_options) { return process_and_run(argv, run) }
+
+      option_rows = File.read(optsfile).split
+      argv.push(*option_rows)
+      argv.slice!(argv.index(optsfile) - 1, 2)
+
```

Comment:
This doesn't do any quoting, splits by any whitespace, including newlines. So may break when passing files with whitespace in them, not sure how to handle that yet.

We append options read onto existing ones, then delete `--option whatever.txt` from ARGV, otherwise we'd get an infinite loop.

Reply:
Maybe require one option per line, and split on the first whitespace (separating option from arg) only?